src/Components/Benefits/Benefits.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react'
 import { Card, CardImg, CardTitle, Button } from 'reactstrap'
 // import Iframe from 'react-iframe'
-import './App.css'
+import './Benefits.css'
```

Comment:
For `.css` file names, use all lowercase. This [discussion](https://stackoverflow.com/a/1614344) explains why.
```suggestion
import './benefits.css'
```